tools/create-service-account.sh

```diff
@@ -1,7 +1,8 @@
 #!/bin/bash
 
+# This script assumes auth is off
 # Create the account
-ID=$(curl  -X POST -H 'Accept: application/json' -H 'Content-Type: application/json' -H 'X-API-Project-Id: USER' -d '{"kind":"service", "name":"service", "uuid":"service"}' 'http://localhost:8080/v1/accounts' | jq -r '.id')
+ID=$(curl  -X POST -H 'Accept: application/json' -H 'Content-Type: application/json' -d '{"kind":"service", "name":"service", "uuid":"service"}' 'http://localhost:8080/v1/accounts' | jq -r '.id')
 
 # Create api keys for the account
 curl  -X POST -H 'Accept: application/json' -H 'Content-Type: application/json' -H "X-API-Project-Id: $ID" -d "{\"accountId\":\"$ID\", \"name\":\"service\", \"publicValue\":\"service\", \"secretValue\":\"servicepass\"}" 'http://localhost:8080/v1/apikeys'
```

Comment:
@cjellick  The X-API-Project-Id: $ID needs to be removed from the create apiKeys curl request as well.